renovate/ts-package-rules.json

```diff
@@ -9,22 +9,22 @@
             "version": "<17"
```

Comment:
The version of Node.js (and `npm`) should be set directly in the `engines` configuration of each repository's `package.json`. Renovate [supports it](https://docs.renovatebot.com/modules/manager/npm) and will automatically update it as well.

Reply:
Of course, it's not directly related to this pull request.

I also took a closer look at Renovate's [`rangeStrategy`](https://docs.renovatebot.com/configuration-options/#rangestrategy). Might be worth adjusting some ranges in a few key repository - like [`config-ci-front`](https://github.com/habx/config-ci-front).